.github/workflows/nightly.yml

```diff
@@ -12,6 +12,16 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@master
 
+      - name: Install hub
+        uses: geertvdc/setup-hub@master
+
+      - name: Delete nightly with hub
+        env:
+          GITHUB_USER: ${{ secrets.GITHUB_USER }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          hub release delete nightly
```

Comment:
what if nightly does not exist?

Reply:
It will fail but it shouldn't block the job to continue.